### PR TITLE
Support Ballerina run command to run jar files 

### PR DESCRIPTION
**Pull request by warunalakshitha**
## Purpose
With this PR, developers can build the jar file for their ballerina project and run with ballerina run command.

ex. 
ballerina build  --jvmTarget  main.bal
ballerina run main.jar

## Check List 
- [x] Read the [Contributing Guide](https://github.com/ballerina-platform/ballerina-lang/blob/master/CONTRIBUTING.md)
- [ ] Required Balo version update
- [ ] Updated Change Log
- [ ] Added necessary tests
  - [ ] Unit Tests
  - [ ] Spec Conformance Tests
  - [ ] Integration Tests
  - [ ] Ballerina By Example Tests
- [ ] Increased Test Coverage   
- [ ] Added necessary documentation  
  - [ ] API documentation 
  - [ ] Module documentation in Module.md files
  - [ ] Ballerina By Examples

### CODE REVIEW

File: tests/ballerina-tools-integration-test/src/test/java/org/ballerinalang/test/run/JarRunFunctionPositiveTestCase.java [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+/**
+ * This class tests invoking main function in a bal for jvm target via Ballerina Build Command and run it the jar for
+ * testing given outputs.
+ */
+public class JarRunFunctionPositiveTestCase extends BaseTest {
```
Comment (Kishanthan) [inline tests/ballerina-tools-integration-test/src/test/java/org/ballerinalang/test/run/JarRunFunctionPositiveTestCase.java]: Is this included in testng?

Comment (warunalakshitha) [reply tests/ballerina-tools-integration-test/src/test/java/org/ballerinalang/test/run/JarRunFunctionPositiveTestCase.java]: yes. This package is already added in tools-integration-test  testng.

File: tests/ballerina-tools-integration-test/src/test/java/org/ballerinalang/test/run/JarRunFunctionPositiveTestCase.java [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+    private Path tempProjectDir;
+
+    private String jarPath = "src/test/resources/run/jar/test_main_for_jvm_target.jar";
```
Comment (Kishanthan) [inline tests/ballerina-tools-integration-test/src/test/java/org/ballerinalang/test/run/JarRunFunctionPositiveTestCase.java]: shall we not create the jar within the resource dir itself? shall we use the target directory to create the jar ?

Comment (warunalakshitha) [reply tests/ballerina-tools-integration-test/src/test/java/org/ballerinalang/test/run/JarRunFunctionPositiveTestCase.java]: Fixed

**Pull request merged**